components/tooltip/README.md

```diff
@@ -0,0 +1,122 @@
+<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
+The tooltip is part of the Gameface custom components suite. As most of the components in this suite it uses slots to allow dynamic content.
+
+Installation
+===================
+
+`npm i coherent-gameface-tooltip`
+
+Usage
+===================
+The tooltip component comes with UMD and CJS builds.
+
+## Usage with UMD modules:
+
+* import the components library:
+
+~~~~{.html}
+<script src="./node_modules/coherent-gameface-components/umd/components.production.min.js"></script>
+~~~~
+
+* import the tooltip component:
+
+~~~~{.html}
+<script src="./node_modules/coherent-gameface-tooltip/umd/tooltip.production.min.js"></script>
+~~~~
+
+* add the tooltip component to your html:
+
+~~~~{.html}
+<gameface-tooltip target=".container">
+    <div slot="message">Hello!</div>
+</gameface-tooltip>
+~~~~
+
+Usually a tooltip is displayed as a result of an action performed on another element. For example you need to show a tooltip with more data when you hover on an item in the inventory. To specify the trigger element use the `target` attribute. Its value is a query string used to get the element. Refer to the table below for the full list of attributes.
+
+
+| Attribute | Type      | Default | Accepted values                  | Description                                                                                      |
+| --------- | --------- | ------- | -------------------------------- | ------------------------------------------------------------------------------------------------ |
+| target       | DOMString | N/A     | Any                              | Specify the element on which the tooltip should be displayed.                                    |
+| on        | String    | click   | `click`, ...                     | Specify the event which will trigger the show method of the tooltip                              |
+| off       | String    | click   | `click`, ...                     | Specify the event which will trigger the hide method of the tooltip                              |

```

Comment:
List all possible accepted values?

Reply:
I'm not sure which events to list.
1. all onevetnt handlers supported by Gameface?
2. custom events?

isn't the list going to be very long?

Reply:
Fair enough, one example is enough to give the clue.